src/MergeContext.js

```diff
@@ -409,13 +485,12 @@ class MergeContext {
                 if (requiredChecks.find(el => el.context.trim() === requiredContext.trim()) === undefined) {
                     // but the passed check matches a required one
```

Comment:
Please note that the following combination of comments does not make sense:

```
// a passed check misses a required one
// but the passed check matches a required one
```
I cannot suggest how to fix this because I am not sure what this all means. Some kind of `context` vs `check` typo?